### PR TITLE
Update versions for node perl python ruby apache subversion git postgres erlang

### PR DESCRIPTION
**Pull request by tw4dl**
@variadico @zquestz

### CODE REVIEW

**Comment by tw4dl** [comment]
6:51] 
+* node-4.4.2
+* perl-5.22.1
+* python-2.7.11
+* python-3.5.1
+* ruby-2.2.4
+* ruby-2.3.0

File: packages/apache-2.4.18.conf [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name:      "apache-2.4.18"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "http://mirror.reverse.net/pub/apache//httpd/httpd-2.4.18.tar.gz",
```
Comment (variadico) [inline packages/apache-2.4.18.conf]: Bug. Two slashes. `//`

Comment (tw4dl) [inline packages/apache-2.4.18.conf]: fixed.

File: packages/apache-2.4.18.conf [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name:      "apache-2.4.18"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "http://mirror.reverse.net/pub/apache/httpd/httpd-2.4.18.tar.gz",
```
Comment (variadico) [inline packages/apache-2.4.18.conf]: https://s3.amazonaws.com/apcera-sources/apache/httpd-2.4.18.tar.gz

File: packages/git-2.8.0.conf [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name:      "git-2.8.0"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "https://www.kernel.org/pub/software/scm/git/git-2.8.0.tar.gz",
```
Comment (variadico) [inline packages/git-2.8.0.conf]: https://s3.amazonaws.com/apcera-sources/git/git-2.8.0.tar.gz

File: packages/postgres-9.4.7.conf [hunk 4]
```diff
@@ -0,0 +1,271 @@
+name:      "postgres-9.4.7"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "https://ftp.postgresql.org/pub/source/v9.4.7/postgresql-9.4.7.tar.gz",
```
Comment (variadico) [inline packages/postgres-9.4.7.conf]: https://s3.amazonaws.com/apcera-sources/postgres/postgresql-9.4.7.tar.gz

File: packages/subversion-1.9.3.conf [hunk 5]
```diff
@@ -0,0 +1,58 @@
+name:      "subversion-1.8.9"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "http://mirror.sdunix.com/apache/subversion/subversion-1.9.3.tar.gz",
```
Comment (variadico) [inline packages/subversion-1.9.3.conf]: https://s3.amazonaws.com/apcera-sources/subversion/subversion-1.9.3.tar.gz

File: runtimes/erlang-otp-18.3.conf [hunk 6]
```diff
@@ -0,0 +1,30 @@
+name:      "erlang-otp-18.3"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "http://erlang.org/download/otp_src_18.3.tar.gz",
```
Comment (variadico) [inline runtimes/erlang-otp-18.3.conf]: https://s3.amazonaws.com/apcera-sources/erlang/otp_src_18.3.tar.gz

File: runtimes/go-1.5.3.conf [hunk 7]
```diff
@@ -0,0 +1,42 @@
+name:      "go-1.5.3"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "https://storage.googleapis.com/golang/go1.5.3.linux-amd64.tar.gz",
```
Comment (variadico) [inline runtimes/go-1.5.3.conf]: https://s3.amazonaws.com/apcera-sources/go/go1.5.3.linux-amd64.tar.gz

File: runtimes/node-4.4.2.conf [hunk 8]
```diff
@@ -0,0 +1,39 @@
+name:      "node"
+version:   "4.4.2"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "https://nodejs.org/dist/v4.4.2/node-v4.4.2.tar.gz",
```
Comment (variadico) [inline runtimes/node-4.4.2.conf]: https://s3.amazonaws.com/apcera-sources/node/node-v4.4.2.tar.gz

File: runtimes/perl-5.22.1.conf [hunk 9]
```diff
@@ -0,0 +1,27 @@
+name:      "perl"
+version:   "5.22.1"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "http://www.cpan.org/src/5.0/perl-5.22.1.tar.gz",
```
Comment (variadico) [inline runtimes/perl-5.22.1.conf]: https://s3.amazonaws.com/apcera-sources/perl/perl-5.22.1.tar.gz

File: runtimes/python-2.7.11.conf [hunk 10]
```diff
@@ -0,0 +1,65 @@
+name:      "python"
+version:   "2.7.11"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "https://www.python.org/ftp/python/2.7.11/Python-2.7.11.tgz",
```
Comment (variadico) [inline runtimes/python-2.7.11.conf]: https://s3.amazonaws.com/apcera-sources/python/Python-2.7.11.tgz

File: runtimes/python-2.7.11.conf [hunk 11]
```diff
@@ -0,0 +1,65 @@
+name:      "python"
```
Comment (variadico) [inline runtimes/python-2.7.11.conf]: @mbhinder @y0ssar1an 

Do these Python package scripts look good to you? Do we also need to update setuptools or virtualenv?

File: runtimes/python-3.5.1.conf [hunk 12]
```diff
@@ -0,0 +1,54 @@
+name:      "python"
+version:   "3.5.1"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz",
```
Comment (variadico) [inline runtimes/python-3.5.1.conf]: https://s3.amazonaws.com/apcera-sources/python/Python-3.5.1.tgz

File: runtimes/python-3.5.1.conf [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+    sha256: "687e067d9f391da645423c7eda8205bae9d35edc0c76ef5218dcbe4cc770d0d7" },
+  { url: "https://pypi.python.org/packages/source/p/pip/pip-8.1.1.tar.gz",
```
Comment (variadico) [inline runtimes/python-3.5.1.conf]: https://s3.amazonaws.com/apcera-sources/python/pip-8.1.1.tar.gz

File: runtimes/ruby-2.2.4.conf [hunk 14]
```diff
@@ -0,0 +1,52 @@
+name:      "ruby"
+version:   "2.2.4"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "https://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.4.tar.gz",
```
Comment (variadico) [inline runtimes/ruby-2.2.4.conf]: https://s3.amazonaws.com/apcera-sources/ruby/ruby-2.2.4.tar.gz

File: runtimes/ruby-2.3.0.conf [hunk 15]
```diff
@@ -0,0 +1,52 @@
+name:      "ruby"
+version:   "2.3.0"
+namespace: "/apcera/pkg/runtimes"
+
+sources [
+  { url: "https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.0.tar.gz",
+    sha256: "ba5ba60e5f1aa21b4ef8e9bf35b9ddb57286cb546aac4b5a28c71f459467e507" },
+  { url: "https://apcera-sources.s3.amazonaws.com/ruby/yaml-0.1.6.tar.gz",
+    sha256: "7da6971b4bd08a986dd2a61353bc422362bd0edcc67d7ebaac68c95f74182749" },
+  { url: "https://rubygems.org/rubygems/rubygems-2.6.2.tgz",
```
Comment (variadico) [inline runtimes/ruby-2.3.0.conf]: https://s3.amazonaws.com/apcera-sources/ruby/rubygems-2.6.2.tgz

File: runtimes/ruby-2.3.0.conf [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+    sha256: "fe9bcd5624579215314e8585852d8214e9be48357e9ab6cb70a594c8a25402c8" },
+  { url: "https://rubygems.org/downloads/bundler-1.11.2.gem",
```
Comment (variadico) [inline runtimes/ruby-2.3.0.conf]: https://s3.amazonaws.com/apcera-sources/ruby/bundler-1.11.2.gem

File: runtimes/python-2.7.11.conf (hunk 11 above)
Comment (mbhinder) [inline runtimes/python-2.7.11.conf]: We probably don't need to but the versions that we are using are pretty old.

Comment (mbhinder) [inline runtimes/python-2.7.11.conf]: The script looks good to me

File: README.md [hunk 17]
```diff
@@ -5,7 +5,7 @@ These are the package config scripts we use to build the runtimes available in t
 Using these definitions can be done through the use of APC:
 
 ```console
-$ apc package build runtimes/go-1.5.1.conf --name go-1.5.1
+$ apc package build runtimes/go-1.5.3.conf --name go-1.5.3
```
Comment (variadico) [inline README.md]: Also, we can remove the `--name go-1.5.3` part. We don't need to pass `--name` anymore.

Comment (tw4dl) [inline README.md]: Here is the command I use to remove old, build new, and export cntmp file: ```
apc package delete package::/apcera/pkg/runtimes::go-1.5.3 --batch ; apc package build runtimes/go-1.5.3.conf --name go-1.5.3 --batch ; apc package export /apcera/pkg/packages::go-1.5.3```

**Comment by tw4dl** [comment]
Thanks @variadico for the s3 urls. Updated to the build scripts.

**Comment by variadico** [comment]
LGTM

**Comment by tw4dl** [comment]
@variadico please also add rsync 3.1.2's gz to s3.

File: packages/apache-tomcat-8.0.33.conf [hunk 18]
```diff
@@ -0,0 +1,29 @@
+name:      "apache-tomcat-8.0.33"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "http://shinyfeather.com/tomcat/tomcat-8/v8.0.33/bin/apache-tomcat-8.0.33.tar.gz",
```
Comment (variadico) [inline packages/apache-tomcat-8.0.33.conf]: https://s3.amazonaws.com/apcera-sources/java/apache-tomcat-8.0.33.tar.gz

File: packages/rsync-3.1.2.conf [hunk 19]
```diff
@@ -0,0 +1,31 @@
+name:      "rsync-3.1.2"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "https://www.samba.org/ftp/rsync/src/rsync-3.1.2.tar.gz",
```
Comment (variadico) [inline packages/rsync-3.1.2.conf]: https://s3.amazonaws.com/apcera-sources/rsync/rsync-3.1.2.tar.gz

File: packages/redis-2.8.24.conf [hunk 20]
```diff
@@ -0,0 +1,28 @@
+name:      "redis-2.8.24"
+namespace: "/apcera/pkg/packages"
+
+sources [
+  { url: "http://download.redis.io/releases/redis-2.8.24.tar.gz",
```
Comment (variadico) [inline packages/redis-2.8.24.conf]: https://s3.amazonaws.com/apcera-sources/redis/redis-2.8.24.tar.gz

**Comment by variadico** [comment]
Josh's tests passed. This is good now.

**Pull request merged**